documentation/src/docs/pravega-concepts.md

```diff
@@ -376,120 +280,108 @@ The following figure depicts the components deployed by Pravega:
+    stream. Readers read events from a stream. Readers are grouped into
+    **Reader Groups** to read from a stream in parallel.
 
--   The Controller is a server-side component that manages the control plane of
-    Pravega.  Streams are created, updated and listed using the Controller API.
+-   The **Controller** is a server-side component that manages the control plane of
+    Pravega.  Streams are created, updated and listed using the controller API.
 
--   The Pravega Server is a server-side component that implements read, write
+-   The **Pravega Server** is a server-side component that implements reads, writes
     and other data plane operations.
 
--   Streams are the fundamental storage primitive in Pravega.  Streams contain a
-    set of data elements called Events.  Events are appended to the “tail” of
-    the Stream by Writers.  Readers can read Events from anywhere in the Stream.
+-   **Streams** are the fundamental storage primitive in Pravega.  Streams contain a
+    set of data elements called events.  Events are appended to the “tail” of
+    the stream by writers.  Readers can read events from anywhere in the stream.
 
--   A Stream is partitioned into a set of Stream Segments. The number of Stream
-    Segments in a Stream can change over time.  Events are written into exactly
-    one of the Stream Segments based on Routing Key.  For any ReaderGroup
-    reading a Stream, each Stream Segment is assigned to one Reader in that
-    ReaderGroup. 
+-   A stream is partitioned into a set of **Stream Segments**. The number of stream
+    segments in a stream can change over time.  Events are written into exactly
+    one of the stream segments based on **Routing Key**.  For any reader group reading a stream, each stream segment is assigned to one reader in that
+    reader group. 
 
--   Each Stream Segment is stored in a combination of Tier1 and Tier2 storage. 
-    The tail of the Segment is stored in Tier1 providing low latency reads and
-    writes.  The rest of the Segment is stored in Tier2, providing high
-    throughput read access with horizontal scalibility and low cost. 
+-   Each stream segment is stored in a combination of **Tier 1** and **Tier 2** storage. 
+    The tail of the segment is stored in Tier 1 providing low latency reads and
+    writes. The rest of the segment is stored in Tier 2, providing high
+    throughput read access with horizontal scalability and low cost. 
 
 ## A Note on Tiered Storage
 
-To deliver an efficient implementation of Streams, Pravega is based on a tiered
-storage model.  Events are persisted in low latency/high IOPS storage  (Tier 1
-Storage) and higher throughput storage (Tier 2 Storage). Writers and Readers are
+To deliver an efficient implementation of streams, Pravega is based on a tiered
+storage model.  Events are persisted in low latency/high IOPS storage (Tier 1
+storage) and higher throughput storage (Tier 2 storage). Writers and readers are
```

Comment:
I would add some more detail to the abstract concept of tiers: `IOPS storage (Tier 1 Storage, write-ahead log) and higher throughput Tier 2 storage (e.g., file system, object store)`.